tests/envs/test_painting.py

```diff
@@ -122,7 +123,8 @@ def test_painting_failure_cases():
     assert OnTable([obj0]) in atoms
     assert OnTable([obj1]) in atoms
     assert OnTable([obj2]) in atoms
-    # In the first initial state, we are holding an object
+    # In the first initial state, we are holding an object, because
+    # painting_initial_holding_prob = 1.0
```

Comment:
lol wait how was this test here before?!

Reply:
this is just a diff on the last commit right?